### ISSUE TITLE
NiceHash QuickMiner

### ISSUE THREAD

**Issue by spudranger** [nicehash/NiceHashQuickMiner#63]
NiceHash QuickMiner - The desktop tray icon disappeared after the first reboot of my machine. How can I get it back?

**Comment by nicehashdev** [comment]
You need to enable Autostart with Windows: https://github.com/nicehash/NiceHashQuickMiner/wiki/Starting-for-the-first-time#4-configuring-nicehash-quickminer

**Comment by spudranger** [comment]
Still have an issue:

I have followed  https://github.com/nicehash/NiceHashQuickMiner/wiki/Starting-for-the-first-time#4-configuring-nicehash-quickminer procedure.

I get the icon appear in the tray taskbar when I run NiceHashQuickMiner for the first time, I have tried both the Task scheduler method and the Service method separately (not at the same time) and on each restart of my computer the Excavator runs with out a problem, **but** I do not get the icon in the tray task bar. The only way I can control the mining is directly through NiceHash webpage, I have no icon. 

How do I get the icon to appear in my tray taskbar in subsequent reboots after the initial startup, everything works on windows startup except I have no icon.
